services/tasks/project/api/routes.py

```diff
@@ -109,7 +109,53 @@ def get_all_tasks():
     }
     return jsonify(response_object), 200
 
+@tasks_blueprint.route('/tasks/update/db', methods=['GET'])
+def update_db():
+    all_repos = current_app.config['REPOS']
 
+    repos = dict()
+    for repo in all_repos:
+        prs = get_items(repo['name'], 'pull_requests')
+        issues = get_items(repo['name'], 'issues')
+        items = issues + prs
+        repos[repo['name']] = items
+
+    for repo in all_repos:
+        if len(repos[repo['name']]) != 0:
+            for item in repos[repo['name']]:
+                if item != None:
+                    task_type = 'pr' if '/pull/' in item['url'] else 'issue'
+                    try:
+                        olditem = Task.query.filter_by(url=item['url']).first()
+                        if olditem is None or olditem.updated_at != updated_at:
+                            db.session.add(
```

Comment:
I believe we need to add an item if `olditem` is `None`, but if it's simply not updated, we need to do something [like this](https://stackoverflow.com/a/6701188).